### DISCUSSION TITLE
Make "Open link in new window" work (or remove option)

### DISCUSSION THREAD

**Discussion by limitedmage** [tauri-apps/tauri#6814] [Q&A]
I'm building a Tauri app on Mac. When you right click a link, a menu shows that includes the option "Open link in new window", but this does not work. How do I either make this work, or remove the context menu item? Thanks!

<img width="374" alt="image" src="https://user-images.githubusercontent.com/206864/234999693-ee692b1c-30ef-484f-8790-2fbcf5d01867.png">

**Comment by xD0135** [comment]
The way I removed the context menu from my sample app is via the following Javascript event:

```
// main.js
document.addEventListener("contextmenu", (event) => event.preventDefault());
```
Note that this disables the context menu from everything in the page, and it may not be what you're looking for but I'm sharing what worked for me :)

**Comment by Scarvy** [comment]
How do I make the context menu options like 'Open Link', 'Open Link in New Window', 'Download Image', and 'Download Image in New Window' work in the context menu? 

<img width="1068" height="592" alt="Screenshot 2025-07-22 at 3 16 33 PM" src="https://github.com/user-attachments/assets/89ea3353-ce1c-47a9-b740-39100af117c3" />

<img width="1438" height="1196" alt="Screenshot 2025-07-22 at 3 18 29 PM" src="https://github.com/user-attachments/assets/fbc02948-9e9e-457e-9a1e-0348b553c893" />

I don't want to disable the native context menu or replace it with frontend code. Is there a way to capture these events and respond to them on the Rust side?

I feel like these should work like 'Copy Image' or 'Copy Link' but they do not. Pleas let know if I'm missing something trivial 🙂
